packages/by-name/microsoft/genpolicy/0001-genpolicy-add-rules-and-types-for-volumeDevices.patch

```diff
@@ -1,7 +1,7 @@
-From 41f26a5803fa50abf3bd0d6cfebc8106ae9dcbc8 Mon Sep 17 00:00:00 2001
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Markus Rudy <mr@edgeless.systems>
 Date: Thu, 23 May 2024 09:20:20 +0200
-Subject: [PATCH 1/6] genpolicy: add rules and types for volumeDevices
+Subject: [PATCH 1/7] genpolicy: add rules and types for volumeDevices
```

Comment:
I did generate them using `-N` and then manually rewrote them since the patch set on main used the `PATCH n/m` formatting. Should I just remove that so only `[PATCH]` is written everywhere?

Reply:
Yes, not all patch sets are converted to the new standard. Please just use `git format-patch` as documented.